redaxo/src/addons/cronjob/lib/manager_sql.php

```diff
@@ -265,6 +289,9 @@ public function saveNextTime($nexttime = null)
         return true;
     }
 
+    /**
+     * @return false|int|null
```

Comment:
```suggestion
     * @return int|null
```

Vermutlich falsch inferred vom tool